main.go

```diff
@@ -56,7 +57,7 @@ func init() {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
-	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-addr", ":8081", "The address the metric endpoint binds to.")
```

Comment:
Not sure I understand this change.

Isn't it tied to the deployment? you can't just change the port in the code, right?
https://github.com/project-koku/korekuta-operator-go/blob/0c17cef1ac360921c9c14e9904fb457fdcd5be11/config/default/manager_auth_proxy_patch.yaml#L24

Reply:
Yea, this was accidental. I had some issue with 8080 and forgot to change it back.